### DISCUSSION TITLE
Guest users from external O365 tenants

### DISCUSSION THREAD

**Discussion by MajinV** [outline/outline#2841] [Self Hosting]
Is it possible to provide access to a user that is not a member or the same O365 tenant?

Currently when they sign in, they effectively just get a new outline instance created for the O365 tenant they are from.

**Comment by tommoor** [comment]
Yes, but they must be invited by email and use email to sign-in rather than O365

**Reply by MajinV** [reply]
Great, will give that a try.

Will only accounts that have specifically invited via email be able to access?

I wonder if there's any legs in looking at multi org support to allow someone in an external directory  to still use their O365 login (with corporate account security etc) across their home outline instance and others they have been explicitly invited to
